packages/opentelemetry-sdk-node/README.md

```diff
@@ -40,10 +40,12 @@ This example uses Jaeger and Prometheus, but exporters exist for [other tracing
 const opentelemetry = require("@opentelemetry/sdk-node");
 const { JaegerExporter } = require("@opentelemetry/exporter-jaeger");
 const { PrometheusExporter } = require("@opentelemetry/exporter-prometheus");
-const { getNodeAutoInstrumentations } = require("@opentelemetry/auto-instrumentations-node");
+const {
+  getNodeAutoInstrumentations,
+} = require("@opentelemetry/auto-instrumentations-node");
 
 const jaegerExporter = new JaegerExporter({
-  serviceName: 'my-service',
+  serviceName: "my-service",
```

Comment:
Was this on purpose as our linter forces single quotes ? (line 45 too)

Reply:
Nope, it's annoying Visual Studio that applies my standard formatting rules one save of files. One of the reason I suggested to define prettier rules in the past to avoid this :)